content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -54,7 +92,4 @@ Refer to the [case study guidelines](https://github.com/cncf/foundation/blob/mas
 
 
 
-## {{% heading "whatsnext" %}}
-
-
-
+## {{% heading "whatsnext" %}}
```

Comment:
Is there anything after this line?

Reply:
No, this was on the initial page so I just left it here. Should I remove it?

Reply:
Not a big deal but nice to remove it.